src/App.tsx

```diff
@@ -27,47 +24,63 @@ function App() {
   const [showLegends, setShowLegends] = useState(true);
   const [pointCount, setPointCount] = useState(0);
   const [mapView, setMapView] = useState<__esri.MapView>();
-  // fetch the data from a MockData.json
-  if (mapView) {
-    // Verify the map is modified
-    fetch("./MockData.json").then((data) => {
-      console.log("Data from MockData.json:", data);
-    });
+  /*
```

Comment:
We used useEffect here to ensure that once the data is successfully loaded from the mapView, 
it triggers the fetching of data from the JSON file and then displays that data in the console.